src/view/prsTreeDataProvider.ts

```diff
@@ -112,6 +113,11 @@ export class PullRequestsTreeDataProvider implements vscode.TreeDataProvider<Tre
 				});
 			}),
 		);
+		this._disposables.push(
+			...reviewModels.map(model => {
+				return model.onDidChangeLocalFileChanges(_ => { this.refresh(); });
```

Comment:
This refreshes the whole tree, instead of just the one PR node. Is this intentional?

Reply:
Calling refresh on the PR node calls refresh on the parent. This propagates all the way up to the tree instance. We discussed this once and decided it would be fine as refreshing only the node created an infinite refreshing state.